src/main/resources/db/migration/V044__Add_validation_warnings_column.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE scannable_items
+ADD COLUMN validationWarnings VARCHAR(1000)[] NULL;
```

Comment:
Field in code is called `ocrValidationWarnings`.
It would be good to keep it consistent I think. The 'ocr' adds clarification, so I'd go with that.

Reply:
I agree it would be better like that. Story description says `validation_warnings` (we don't use underscored by convention, unfortunately). @nitinprabhu as you proposed current name, would you be ok with adding `ocr` to the beginning?